### PR TITLE
Active call endpoints

### PR DESCRIPTION
**Pull request by mmadariaga**
Expose platform/brand/company active calls through API

### CODE REVIEW

File: web/rest/brand/src/Controller/My/ActiveCallsAction.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            return new ActiveCalls($activeCalls);
+        }
+
+        /** @var CompanyInterface | null $company */
+        $company = $this->companyRepository->find($companyId);
+        if (!$company) {
+            throw new NotFoundHttpException('Company not found');
+        }
+
+        if ($company->getBrand() !== $brand) {
```
Comment (Kaian) [inline web/rest/brand/src/Controller/My/ActiveCallsAction.php]: Is this better than comparing brand Ids?

File: web/rest/brand/app/config/api/resources.yml [hunk 2]
```diff
@@ -50,3 +50,24 @@ Model\RatingPlanPrices:
             description: 'Rating plan group Id'
   attributes:
     pagination_client_enabled: true
+
+Model\ActiveCalls:
+  itemOperations:
+    get_my_active_calls:
+      access_control: '"ROLE_BRAND_ADMIN" in roles'
+      method: 'GET'
+      path: '/my/active_calls'
```
Comment (Kaian) [inline web/rest/brand/app/config/api/resources.yml]: Out of curiosity, will be _my_ the _namespace_ for all custom endpoints?  Or it's also possible to use _Provider_?

Comment (mmadariaga) [reply web/rest/brand/app/config/api/resources.yml]: There is no technical limitation about namespaces

File: web/rest/brand/src/Controller/My/ActiveCallsAction.php (hunk 1 above)
Comment (mmadariaga) [reply web/rest/brand/src/Controller/My/ActiveCallsAction.php]: Doctrine uses the same instance for the same id, so the result would not change. It's is just shorter

Comment (Kaian) [reply web/rest/brand/src/Controller/My/ActiveCallsAction.php]: Oh, i've been comparing ids all my life thinking it was safer :(

**Comment by ironArt3mis** [comment]
Rebase completed.

**Pull request merged**